### PR TITLE
helm - kibana ingress path fix

### PR DESCRIPTION
**Pull request by Hokwang**
Hi,

this is for fixing #307

I did
- user can input multiple hosts -> user can input only one host
- fix ingress path
- etc

### CODE REVIEW

File: helm/opendistro-es/templates/kibana/kibana-ingress.yml [hunk 1]
```diff
@@ -13,33 +13,25 @@
...
 {{- if and .Values.kibana.ingress.enabled .Values.kibana.enabled }}
-{{- $serviceName:= printf "%s-%s"  (include "opendistro-es.fullname" .) "kibana-svc" }}
-{{- $servicePort := .Values.kibana.externalPort }}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  labels:
-{{ include "opendistro-es.labels.standard" . | indent 4 }}
-  name: {{ template "opendistro-es.fullname" . }}-kibana
+  labels: {{ include "opendistro-es.labels.standard" . | nindent 4 }}
```
Comment (peterzhuamazon) [inline helm/opendistro-es/templates/kibana/kibana-ingress.yml:19]: Is "nindent" a typo right here?

**Comment by peterzhuamazon** [comment]
Hi @Hokwang thanks for your contribution.
Have you test this setup in your local and success?
Thanks.

**Comment by Hokwang** [comment]
edit one more for minimal changes
and I've tested. there's no problem.

**Comment by peterzhuamazon** [comment]
Thanks @Hokwang this PR is now merged to master.

**Pull request merged**